frontend/src/container/Trace/Filters/Panel/PanelBody/Common/styles.ts

```diff
@@ -9,3 +10,10 @@ export const CheckBoxContainer = styled.div`
 	margin-top: 0.5rem;
 	margin-bottom: 0.5rem;
 `;
+
+export const ParaGraph = styled(Typography.Paragraph)`
+	&&& {
+		margin: 0;
+		max-width: 8rem;
```

Comment:
IMO this should be some percentage of screen size (20% or something). Because on bigger screen 8rem is very small.

Reply:
let me check

Reply:
@pranshuchittora as the parent as width of 15rem

Reply:
https://stackoverflow.com/questions/13941275/how-does-rem-differ-from-em-in-css#:~:text=While%20em%20is%20relative%20to,across%20the%20entire%20page%20easily.

Reply:
Can you pls check this and do a quick POC in codepen or something

Reply:
this would be big change if we change the base font size in html _(with Axolo)_

Reply:
not sure what is the purpose of em vs rem _(with Axolo)_

Reply:
Percentage should be the right unit. Rem won’t scale with paren’t width _(with Axolo)_

Reply:
let me check this one _(with Axolo)_

Reply:
actually rem take the reference of the base font set in the html.. but this won't take any effect has i have restricted the _(with Axolo)_

Reply:
parent width here _(with Axolo)_

Reply:
but yes to make this flexible % would be right _(with Axolo)_